<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="QC-VuHoang" ignored="1" name="Build 001" tests="1" failures="1" timestamp="04 Apr 2019 05:41:25 GMT" time="11.3" errors="0">
  <testcase name="TC001" time="2.163" classname="tests.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [Số điện thoại hoặc mật khẩu không đúng, vui lòng đăng nhập lại!] but found [Số điện thoại hoặc mật khẩu không đúng, vui lòng đăng nhập lại.]">
      <![CDATA[java.lang.AssertionError: expected [Số điện thoại hoặc mật khẩu không đúng, vui lòng đăng nhập lại!] but found [Số điện thoại hoặc mật khẩu không đúng, vui lòng đăng nhập lại.]
at utils.SeleniumSupport.verifyDisplayWithText(SeleniumSupport.java:188)
at modules.LoginPage.verify_error_message(LoginPage.java:28)
at tests.LoginTest.TC001(LoginTest.java:20)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- TC001 -->
  <testcase name="TC002" time="0.0" classname="tests.LoginTest">
    <ignored/>
  </testcase> <!-- TC002 -->
</testsuite> <!-- Build 001 -->
